libs/barista-components/filter-field/src/filter-field.ts

```diff
@@ -516,6 +518,18 @@ export class DtFilterField<T = any>
   /** Whether the filter field or one of it's child elements is focused. */
   private _isFocused = false;
 
+  /** Whether to allow focusing the tags when navigating in the input element using arrow keys */
+  private _allowArrowKeyFocusOnTags = false;
+
+  private _currentFocusState: 'none' | 'input' | 'inputboundry' = 'none';
```

Comment:
Please add jsDoc here as as well as in line 526.
```suggestion
  private _currentFocusState: 'none' | 'input' | 'inputboundary' = 'none';
```